spec/commands/feed/import_from_opml_spec.rb

```diff
@@ -1,104 +1,111 @@
 # frozen_string_literal: true
 
 RSpec.describe Feed::ImportFromOpml do
-  let(:subscriptions) do
-    File.open(
-      File.expand_path(
-        "../../support/files/subscriptions.xml",
-        __dir__
-      )
-    )
+  tmw_football = {
+    name: "TMW Football Transfer News",
+    url: "http://www.transfermarketweb.com/rss"
+  }
+
+  giant_robots = {
+    name: "GIANT ROBOTS SMASHING INTO OTHER GIANT ROBOTS - Home",
+    url: "http://feeds.feedburner.com/GiantRobotsSmashingIntoOtherGiantRobots"
+  }
+
+  autoblog = { name: "Autoblog", url: "http://feeds.autoblog.com/weblogsinc/autoblog/" }
+  city_guide = { name: "City Guide News", url: "http://www.probki.net/news/RSS_news_feed.asp" }
+
+  def create_subscriptions(
+    file: "../../support/files/subscriptions.xml",
+    path: __dir__
+  )
```

Comment:
In terms of naming, this seems more like a `subscriptions` or `read_subscriptions`. Also, I don't see `file` or `path` being passed in anywhere, so maybe they can be hard coded in the method?

```suggestion
  def read_subscriptions
```